server.js

```diff
@@ -22,9 +17,80 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+// --- ALL THE ROUTES ---
+
+// --- front page ---
+app.get('/', (request /* incoming */, response /* outgoing */) => {
```

Comment:
These inline comments make it look a little messy 🙈 Maybe keep this in your personal notes instead 😇